requirements-base.txt

```diff
@@ -1,6 +1,6 @@
 beautifulsoup4>=4.7.1,<4.8
-boto3>=1.4.1,<1.4.6
-botocore<1.5.71
+boto3>=1.10.1,<1.11
+botocore<1.14.0
```

Comment:
I have a separate PR to bump up the versions: https://github.com/getsentry/sentry/pull/22543